pkg/provisioner/ironic/raid.go

```diff
@@ -172,3 +226,21 @@ func BuildRAIDCleanSteps(raid *metal3v1alpha1.RAIDConfig) (cleanSteps []nodes.Cl
 	)
 	return
```

Comment:
Still on the troubleshooting side, do we have any way to easily understand which clean steps are going to be applied? If not, I'd suggest to add log trace here https://github.com/metal3-io/baremetal-operator/blob/62372c1daa01c631da46109078c9872128088f1e/pkg/provisioner/ironic/ironic.go#L1068 for that

Reply:
Here will print the current clean step:
https://github.com/metal3-io/baremetal-operator/blob/62372c1daa01c631da46109078c9872128088f1e/pkg/provisioner/ironic/ironic.go#L1029-L1041

Reply:
Fine, maybe for consistency we could keep the log in both points

Reply:
Done.